ingestion/src/metadata/ingestion/source/dashboard/metabase/connection.py

```diff
@@ -70,8 +51,8 @@ def test_connection(
 
     def custom_executor():
         result = requests.get(  # pylint: disable=missing-timeout
-            client["connection"].hostPort + "/api/dashboard",
-            headers=client["metabase_session"],
+            client.host_port + "/api/dashboard",
+            headers=client.metabase_session,
         )
 
         return list(result)
```

Comment:
The client of this method type should be `MetabaseClient,` and this code must be replaced for:
```python
return client.get_dashboards_list()
```
wdyt?